travis/stage-python3.sh

```diff
@@ -7,7 +7,7 @@ then
   ./travis/test-anaconda-compatibility.sh "anaconda3:2019.03"
 fi
 CHANGED_FILES=$(git diff --name-only master..HEAD | grep "tests/examples\|examples") || true
-if [[ "$TRAVIS_EVENT_TYPE" == "cron" || "$CHANGED_FILES" == *"examples"* ]] && [[ "$TRAVIS_BUILD_STAGE_NAME" == "Nightly" ]]
+if [[ "$TRAVIS_EVENT_TYPE" == "cron" || "$CHANGED_FILES" == *"examples"* ]]
```

Comment:
Fix for running the example tests on pull requests

Reply:
We can remove the check for Dockerfiles at [line 14](https://github.com/mlflow/mlflow/blob/74f725da4a4464cb20437ed8ecf0aa45f5c55d22/travis/stage-python3.sh#L14) as well